core/src/commonMain/kotlin/com/xebia/functional/xef/auto/AI.kt

```diff
@@ -58,7 +56,14 @@ suspend fun <A> MockAIScope(
   try {
     val embeddings = OpenAIEmbeddings(OpenAIConfig(), mockClient)
     val vectorStore = LocalVectorStore(embeddings)
-    val scope = AIScope(mockClient, vectorStore, embeddings)
+    val scope =
```

Comment:
Not to address in this pull request, but we mentioned moving out the `MockAIScope ` to a separate `test` module

Reply:
Once we have a local model in the ai DSL, we should not have these mocks but true integration tests against our model of choice.